README.md

```diff
@@ -1113,6 +1113,37 @@ In either case:
        an attribute when `self` is an ActiveRecord model: `self.guest = user`.
     3. Referencing the current instance's class: `self.class`.
 
+* <a name="freeze-constants"></a>When defining arrays and hashes meant to be
+    constants, make sure to call `freeze` on them. ([More on this][ruby-freeze].)
+    <sup>[[link](#freeze-constants)]</sup>
+
+    The reason is that Ruby constants are actually mutable. Calling `freeze`
+    ensures they are not mutated and are therefore truly constant and 
+    attempting to modify them will raise an exception. 
+
+    ```ruby
+    class COLOR
+      # Each color
+      RED = 0
```

Comment:
Searching in the code, it looks like we're mainly doing it where performance has bitten us, and not as a general practice. 

For a lot of the missing ones, it feels like symbols would be better anyway. So perhaps we can have some guidelines around when to prefer symbols and then add that string constants should be frozen. Or it could be a separate guideline linked from this one too since it is a thing of its own.